### DISCUSSION TITLE
WINMERGE CAPABILITY

### DISCUSSION THREAD

**Discussion by silfoto** [WinMerge/winmerge#1561] [Q&A]
Can WinMerge do this?.....
I’m looking for a way to MOVE multiple FOLDERS and all of each of its multilevel subfolders and their SUBFOLDERS of every level with each of their contained files to a new single destination folder?   In other words….I want all of the original files and folders to be moved, to a single destination folder with all of the source folders moved to the 1st Level Destination Folder?  See below for clarification of what I am asking.  I have hundreds of thousand files in a hundred to two hundred or so top-level folders that I want to move in this fashion to a dozen or so new destination folders. I have over 11 Million files to move with approx. 3TB in total size.  Not sure how many top-level source files but safe to assume thousands. 
 
If your WinMerge program cannot do this, I’d appreciate any leads on other ways to accomplish it.
![FOLDER MERGE DIAGRAM 2022-11-06 170359](https://user-images.githubusercontent.com/52113409/200206594-42a77e16-c7c2-4a2e-9af0-72dc0876f097.png)

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have the right features for what you want to do.

The following URL may be helpful

https://www.winhelponline.com/blog/move-copy-files-multiple-sub-folders-single-folder/

**Comment by silfoto** [comment]
Thank you. This article looks promising. Much appreciated!
________________________________
